src/Tribe/Admin/Provider.php

```diff
@@ -15,6 +15,7 @@ public function register() {
 		$this->container->singleton( 'tec.admin.notice.timezones', Notice\Timezones::class );
 		$this->container->singleton( 'tec.admin.notice.marketing', Notice\Marketing::class );
 		$this->container->singleton( 'tec.admin.notice.update', Notice\Update::class );
+		$this->container->singleton( 'tec.admin.notice.install-event-tickets', Notice\Install_Event_Tickets::class );
```

Comment:
Lets go with:
```suggestion
		$this->container->singleton( Notice\Install_Event_Tickets::class, Notice\Install_Event_Tickets::class );
```

We should be avoiding the string notation where we can.